stage1/init/common/pod.go

```diff
@@ -454,6 +454,49 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		unit.NewUnitOption("Service", "SyslogIdentifier", appName.String()),
 	}
 
+	// Restrict access to some security-sensitive paths under /proc and /sys.
+	// Those entries can be hidden or just made read-only to app.
+	restrictedPaths := []struct {
+		path         string
+		inaccessible bool
+		readonly     bool
+	}{
```

Comment:
Since `inaccessible` and `readonly` are mutually exclusive, and the for loop handle them separately, `restrictedPaths` could be split in two separate arrays of strings.